uaaextras/templates/email/body.html

```diff
@@ -17,18 +17,18 @@
         <p>
         <ol>
             <li>
-            <b>Accept the invitation</b> - <a href="{{verification_url}}">Accept your invite</a> to continue the registration process. (The invitation will expire after 24 hours.) You can also copy the URL below and paste it into your browser's address bar:
+            <b>Accept the invitation</b> - <a href="{{verification_url}}">Accept your invite [2]</a> to continue the registration process. (The invitation will expire after 24 hours.) You can also copy the URL below and paste it into your browser's address bar:
             <br />
             {{verification_url}}
```

Comment:
Nothing currently displays. Is that known/intended behavior? Do we have a check that prevents this email from being sent (or from including this link) if this particular URL has no value?

Reply:
Same point about the local variables.